waspc/src/Wasp/Analyzer/Parser/Tokenizer.hs

```diff
@@ -0,0 +1,26 @@
+module Wasp.Analyzer.Parser.Tokenizer
```

Comment:
Should I rename this file to something like `LexerUtils`? Or should I perhaps put some (of all) of the functions defined here inside `Lexer.x` and import them using `import Wasp.Analyzer.Parser.Lexer (..)`?

Reply:
Ha that is a good question! I wouldn't move this stuff to Lexer.x, I would rather keep Lexer.x light. We could maybe rename it to LexerUtils though. But I am not sure! Maybe leave it as Tokenizer but comment on it?

Reply:
Perhaps we can expose all the lexer stuff through the same file, something like [what JavaScript can do](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export#re-exporting_aggregating). In other words, does Haskell have a nicer equivalent for something like this (and is this even legal):
```haskell
module Wasp.Analyzer.Parser.Lexer (lexer, tokenize, isValidWaspIdentifier) where

-- instead of Lexer.x, we would have AlexLexer.x
import Wasp.Analyzer.Parser.AlexLexer (lexer, alexScanTokens)

isValidWaspIdentifier :: String -> Bool
isValidWaspIdentifier = -- ...

tokenize :: String -> Maybe [Token]
tokenize = -- ... uses 'alexScanTokens'
```
Then we could always import everything lexer-related from `Wasp.Analyzer.Parser.Lexer` without having to cram all the lexer utilities inside Happy's DSL:
```haskell
-- In Wasp.Cli.Command.CreateNewProject
import Wasp.Analyzer.Parser.Lexer (isValidWaspIdentifier)

-- In Parser.y (unless Parser.y actually requires us to import everything?)
import Wasp.Analyzer.Parser.Lexer (lexer)
```